### DISCUSSION TITLE
LayerNormKernelImpl" not implemented for 'Half'

### DISCUSSION THREAD

**Discussion by ExecuteAI** [vladmandic/sdnext#464] [Q&A]
Is there anyone besides Vlad who is busy who can help me with this question?
I have made the path to the windows/system32 for the nvidia-smi file, and it worked for a while.
But the engine won't start, it showed me the error for “half” 

![image](https://user-images.githubusercontent.com/79231405/234106000-961f0554-ce01-4be1-80d7-0ee2bc3e5747.png)

**Comment by ExecuteAI** [comment]
![image](https://user-images.githubusercontent.com/79231405/234106078-5e941871-5bf9-4746-a2e4-01b33093d63a.png)

**Reply by ExecuteAI** [reply]
Can anyone explain to me why the CUDA is not detected when I have a 2080 TI, and it just did, now it doesn't?

**Comment by ExecuteAI** [comment]
And this one? RuntimeError: "log_vml_cpu" not implemented for 'Half'

**Comment by ExecuteAI** [comment]
![image](https://user-images.githubusercontent.com/79231405/234108393-40ddfa64-b48e-4498-865c-327c3260ac29.png)

**Comment by ExecuteAI** [comment]
Does anybody know where I change this? RuntimeError: "log_vml_cpu" not implemented for 'Half'
I don't know about PyTorch or any other python programming. I can't change my webui.bat, then I get an error. 
Any other file I change, I get an error. 
I'm stuck and so close to making it work.

**Comment by ExecuteAI** [comment]
I did make a path to the windows system32 for the nvidia-smi file and it worked for a while. Now I get the same error.
![image](https://user-images.githubusercontent.com/79231405/234111697-825a2c5f-0636-4815-a9d8-5624b3fe574a.png)

**Comment by vladmandic** [comment]
if on startup it says using torch+cpu, that means that wrong torch is installed. you added currect path, but torch is already installed and its not going to replace itself. you need to uninstall it and let setup install correct one. focus on getting correct torch, stop focusing on errors that would be relevant only if you actually wanted to run on cpu.
